### PR TITLE
December Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | bbb47a5 |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-374.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20231215_amplify.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+#### Known issues for Asset Catalog
+
+There are no known limitations in this release.
+
+### Product Foundry
+
+#### Feature updates for Product Foundry
+
+* **Limit the product plan time** - new option is available when creating a paid plan to limit the number or renewal (unlimited by default). Like this is possible for the consumer to subscribe for a limited period of time and be sure at the end of the contract that the subscription will be terminated.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20231215_amplify.md:157]: Configure a plan term lenght -  a new option is now available when creating / editing a plan to allow providers to configure how often a subscription to this plan will renew (unlimited by default). When a subscription reaches the term lenght configured on the plan, the subscription will be automatically cancelled.

File: content/en/docs/amplify_relnotes/20231215_amplify.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+## Consumer experience
+
+### Marketplace
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **Limit the product plan time** - When viewing a plan, the number of term for which the plan can be used is display in the plan details view. The subscription details screen has been reorganized to show when the subscription will/has expired.
+* **Remove corrupted resources from Active plan** - when an API Service is removed but was used in a product plan, the product plan becomes corrupted. It is now possible when viewing the plan quota to remove the corrupted resources using the in icon. The bin icon is also present when the plan is in draft state.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20231215_amplify.md:174]: remove this sentence: when an API Service is removed but was used in a product plan, the product plan becomes corrupted.

Comment (georgiana-caras) [reply content/en/docs/amplify_relnotes/20231215_amplify.md:174]: using the **in** icon. I assume it wanted to say: bin icon

File: content/en/docs/amplify_relnotes/20231215_amplify.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+* **Subscription approval events get sent twice** - Now only one approval event is sent.
+
+#### Bug fixes for Marketplace
+
+* **HTTP Basic not available when authorizing** - With the introduction of the new *Authentication* side panel, it is now possible, for any type of credential, to test the API by providing values that have not been provision from Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20231215_amplify.md:179]: The **HTTP Basic** auth type has been added to the Authentication side pane.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Pull request merged**